packages/devtools-reps/src/object-inspector/tests/utils/make-node-for-properties.js

```diff
@@ -0,0 +1,258 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const {
+  isDefault,
+  makeNodesForProperties,
+} = require("../../utils");
+
+const root = {
+  path: "root",
+  contents: { value: {} }
+};
+
+const objProperties = {
+  ownProperties: {
+    "0": {
+      value: {}
+    },
+    "2": {},
+    length: {
+      value: 3
+    }
+  },
+  prototype: {
+    type: "object",
+    actor: "server2.conn1.child1/pausedobj618",
+    class: "bla"
+  }
+};
+
+describe("makeNodesForProperties", () => {
+  it("kitchen sink", () => {
+    const nodes = makeNodesForProperties(objProperties, root);
+
+    const names = nodes.map(n => n.name);
+    expect(names).toEqual(["0", "length", "__proto__"]);
+
+    const paths = nodes.map(n => n.path);
+    expect(paths).toEqual(["root/0", "root/length", "root/__proto__"]);
+  });
+
+  it("includes getters and setters", () => {
+    const nodes = makeNodesForProperties(
+      {
+        ownProperties: {
+          foo: { value: "foo" },
+          bar: {
+            "configurable": true,
+            "enumerable": true,
+            "get": {
```

Comment:
this is minor, but do we need all these fields for the test to be valid?

I like small fixture data because it's easier to see what the data dependencies are.